cmd/antrea-controller/controller.go

```diff
@@ -142,6 +158,7 @@ func createAPIServerConfig(kubeconfig string,
 		return nil, err
 	}
 
+	serverConfig.LoopbackClientConfig.BearerToken = generatedTokenString
```

Comment:
I think `secureServing.ApplyTo` has generated one token and you don't have to generate the token and set it?

Reply:
Thanks, I didn't notice before, updated.